db/structure.sql

```diff
@@ -2,15 +2,14 @@
 -- PostgreSQL database dump
 --
 
--- Dumped from database version 9.6.5
--- Dumped by pg_dump version 9.6.8
+-- Dumped from database version 10.1
+-- Dumped by pg_dump version 10.1
```

Comment:
Please only use pg_dump of 9.6. If you don't the dump files don't match and merge chaos ensues.

Reply:
As an aside, the docker-compose.yml uses the correct versions.